cinema_project/cinema_project/settings.py

```diff
@@ -142,3 +143,5 @@
 LOGOUT_REDIRECT_URL = "/"
 
 CONTACT_EMAIL = env("CONTACT_EMAIL")
+
+IMPORT_EXPORT_USE_TRANSACTIONS = True
```

Comment:
This is really cool and seems super easy to use! I didn't know about it, so thanks!
As an exercise, it would have been better to process the csv and do the insert/updates in some method(s) you wrote, though.
Also, maybe this needs a line in the requirements file.

Reply:
Change external library usage to built-in for import functionality